app/src/androidTest/java/com/courseproject/tindar/ds/DatabaseHelperTest.java

```diff
@@ -60,6 +59,8 @@ public void setUp() {
 
     @After
     public void tearDown() {
+        dbHelper.deleteLikeLists();
+        dbHelper.deleteAccounts();
```

Comment:
these two lines no longer needed and should be deleted. Now we have dbHelper.deleteAllDbRecords() inside setup to clean out the db records

Reply:
done